### PR TITLE
[DLS-6789] Adding Navigation To Packing Site Details

### PR DESCRIPTION
**Pull request by JakeReid2020**

### CODE REVIEW

File: app/utilitlies/UserTypeCheck.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+package utilitlies
+
+import models.SdilReturn
+import models.retrieved.RetrievedSubscription
+
+
+object UserTypeCheck{
+def isNewImporter(sdilReturn:SdilReturn,subscription:RetrievedSubscription):Boolean ={
```
Comment (ChrisHillCap) [inline app/utilitlies/UserTypeCheck.scala]: werent we going to put on checking the warehousedetails on the end?

File: app/utilitlies/UserTypeCheck.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * limitations under the License.
+ */
+
+package utilitlies
+
+import models.SdilReturn
+import models.retrieved.RetrievedSubscription
+
+
+object UserTypeCheck{
```
Comment (ChrisHillCap) [inline app/utilitlies/UserTypeCheck.scala]:
```suggestion
object UserTypeCheck {
```

File: app/utilitlies/UserTypeCheck.scala (hunk 1 above)
Comment (ChrisHillCap) [inline app/utilitlies/UserTypeCheck.scala]:
```suggestion

def isNewImporter(sdilReturn: SdilReturn,subscription: RetrievedSubscription): Boolean = {
```

File: it/controllers/testSupport/ITCoreTestData.scala [hunk 3]
```diff
@@ -110,6 +110,16 @@ trait ITCoreTestData extends TryValues {
     .set(BroughtIntoUkFromSmallProducersPage, false).success.value
     .set(ClaimCreditsForExportsPage, false)
 
+  def newPackerPartialNewImporterAnswers = emptyUserAnswers
```
Comment (ChrisHillCap) [inline it/controllers/testSupport/ITCoreTestData.scala:113]: do we need all these values? cant we just have the answers contain the things that are required on the page?
cant we just set the answers that are needed on the page to do the test

**Review by ChrisHillCap — CHANGES_REQUESTED** [review]

**Review by JayenKerai — APPROVED** [review]

**Review by ChrisHillCap — APPROVED** [review]

**Pull request merged**